controllers/controller.go

```diff
@@ -0,0 +1,78 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/gror/model"
+	"github.com/gror/services"
+	"gopkg.in/mgo.v2/bson"
+)
+
+func CreateDocker(w http.ResponseWriter, r *http.Request) {
+
+	//defer r.Body.Close()
+	var rootobject model.Root
+	err := json.NewDecoder(r.Body).Decode(&rootobject)
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+	rootobject.ID = bson.NewObjectId()
+
+	marshalData, _ := json.Marshal(rootobject)
+
+	err = services.UnmarshalInsertData(marshalData)
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(201)
+	fmt.Fprintf(w, "%s", marshalData)
```

Comment:
Instead of printing the data here, return valid http status code indicating if its a success or failure